lib/osf-components/addon/components/registries/registration-form-navigation-dropdown/component.ts

```diff
@@ -14,6 +16,14 @@ export default class RegistrationFormNavigationDropdown extends Component {
     // Required parameters
     schemaBlocks!: SchemaBlock[];
 
+    // Optional paramaters
+    showMetadata?: boolean = false;
```

Comment:
```suggestion
    showMetadata: boolean = false;
```
you don't actually need the type either, but ¯\\\_(ツ)\_/¯